### DISCUSSION TITLE
［委托］高价收取青衣小姐喝剩的茶叶和吐出来的茶渣喵

### DISCUSSION THREAD

**Discussion by abyssal-sea-c** [share121/inter-knot#5457] [委托]
高价回收青衣小姐喝剩的茶叶喵
有意者请联系0721onani@0721yuuu喵
![illust_122748630_20240928_150127](https://github.com/user-attachments/assets/f5718cb9-5fb7-49ad-8083-89b1a5b5d9c2)

**Comment by KELERSAMA** [comment]
我愿意比楼主多出一块

**Comment by fzdqz** [comment]
你等会我去问下青衣

**Comment by nishidsb** [comment]
我问了青衣，她说卖你了我就没饭了
